Dockerfile

```diff
@@ -20,4 +20,6 @@ RUN echo 'APT::Install-Recommends "0" ; APT::Install-Suggests "0" ;' > /etc/apt/
     apt-get clean && apt-get --yes --quiet autoremove --purge && \
     rm -rf /var/lib/apt/lists/* /var/log/* /tmp/* /usr/share/locale/*
 
-RUN pip3 install -U sphinx
+RUN pip3 install --upgrade setuptools wheel
+RUN pip3 install --upgrade sphinx
```

Comment:
Could also install by apt to be faster (to be set before the apt-get clean)
apt-get install python3-sphinx
apt-get install python3-setuptools
apt-get install python3-wheel